assigner.py

```diff
@@ -35,24 +37,119 @@ def main():
     else:
         logger.setLevel(logging.INFO)
 
-    # Do things!
     try:
         with open(args.configfile) as fd:
             config = yaml.load(fd)
     except Exception as e:
         logger.critical("Could not parse configuration file: {}".format(e))
         sys.exit(127)
+
+    # Bugzilla setup
     bcfg = config.get('bugzilla')
-    bapi = bugzilla.Bugzilla(url=bcfg.get('url'), api_key=os.environ.get('BUGZILLA_API_KEY'))
+    bapi_key = os.environ.get('BUGZILLA_API_KEY')
+    if (bapi_key is None):
+        logger.critical("No Bugzilla API Key passed in environment variable BUGZILLA_API_KEY")
+        sys.exit(127)
+    bapi = bugzilla.Bugzilla(url=bcfg.get('url'), api_key=bapi_key)
+
+    # Casa setup
+    ccfg = config.get('casa')
+    capi_key = os.environ.get('CASA_API_KEY')
+    if (capi_key is None):
+        logger.critical("No CASA API Key passed in environment variable CASA_API_KEY")
+        sys.exit(127)
+    capi = casa.Casa(url=ccfg.get('url'), api_key=capi_key)
 
+    # Do things!
     autoassign(bapi, bcfg.get('rra'), args.dry_run)
     autoassign(bapi, bcfg.get('va'), args.dry_run)
+    autocasa(bapi, capi, bcfg, ccfg, args.dry_run)
 
+def autocasa(bapi, capi, bcfg, ccfg, dry_run):
```

Comment:
I wonder if it's better to de-couple CASA and generic BMO service assignment logic into two different scripts.  Basically, what I was suggesting here => https://github.com/mozilla/infosec-risk-management-bugzilla/issues/6

Reply:
i started with separate scripts, but since all configuration is shared and there is a depency issue i ended up integrating it